### ISSUE TITLE
The reward cannot converge, I've tried in different computer, the training results of the paper cannot be reproduced? Could you give me some suggestions?

### ISSUE THREAD

**Issue by mrayur** [anh-nn01/RL4Suspension-ICMLA23#1]
![Body Acceleration](https://github.com/user-attachments/assets/c71df50e-156d-423b-b2d4-16c0c0f89b62)
![Body displacement](https://github.com/user-attachments/assets/744f355e-6133-49e1-aa7c-f19b838334b1)
![Body Velocity](https://github.com/user-attachments/assets/131b26e2-2627-4478-8427-6415969427b3)
![Damping](https://github.com/user-attachments/assets/c1b868cc-9e5c-4e54-82b5-f3ab4f9d435c)
![Road Excitation](https://github.com/user-attachments/assets/8a1bdddb-32fc-452f-b210-b49ccbd429ab)
![Spring Stiffness](https://github.com/user-attachments/assets/298c2083-1a5f-468e-a2d6-b0f6d9393452)

Hi,

Thanks for your valuable work, but I cannot reproduce the results in your paper. Have you met the same problem while developing the project? I've tried more 10 time on my computer and laptop, all failed to make the reward converge. Could you give me some tips?

Looking forward to your early reply! Thanks in advance.

Best,
John

**Comment by mrayur** [comment]
By the way, I used the cpu to train the neural network.

**Comment by LuchySweetRain** [comment]
@mrayur Hello, have you solved the issue of the reward not converging? I found that while the reward does not converge during training, the suspension control does have some effect, for example, the body displacement has decreased. I don't understand why this is happening.
